### PR TITLE
Preserve func.yaml for Quarkus BP build

### PR DESCRIPTION
**Pull request by matejvasek**
# Changes

- :bug: Fix On Cluster Build issues with Quarkus buildpack build not preserving `func.yaml`.

```release-note
bugfix: on cluster build issues with Quarkus buildpack build not preserving `func.yaml`
```

### CODE REVIEW

**Comment by matejvasek** [comment]
Beware I did not tested this code with Tekton Pipeline.

I just tested that an image build locally does contain the `func.yaml` file, based on that I believe this should fix in cluster build for Quarkus.

**Review by lance — COMMENTED** [review]
/lgtm
/hold for others/tests
Funny how sometimes it's only a couple of simple changes that are needed! 👍

**Comment by matejvasek** [comment]
@lance I also updated validation, please re-lgtm.

File: pipelines/tekton/validate_test.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ func Test_validatePipeline(t *testing.T) {
 		{
 			name:     "Unsupported runtime - Quarkus - without additional Buildpacks",
```
Comment (lance) [inline pipelines/tekton/validate_test.go]: Nit
```suggestion
			name:     "Supported runtime - Quarkus - without additional Buildpacks",
```

**Review by lance — COMMENTED** [review]
/lgtm with one nit

**Pull request merged**